### PR TITLE
Fix: Throw error if path to tsconfig.json is incorrect

### PR DESCRIPTION
**Pull request by harsha509**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-runner/src/index.ts [hunk 1]
```diff
@@ -102,6 +102,9 @@ export default class Runner extends EventEmitter {
          * autocompile after parsing configs so we support ES6 features in tests with config driven by users
          */
         if (args.autoCompileOpts?.autoCompile) {
+            if (!fs.existsSync(args.autoCompileOpts.tsNodeOpts.project))
```
Comment (christian-bromann) [inline packages/wdio-runner/src/index.ts:105]:
```suggestion
            if (!fs.existsSync(args.autoCompileOpts?.tsNodeOpts?.project))
```
```suggestion
            if (!fs.existsSync(args.autoCompileOpts.tsNodeOpts.project))
```

File: packages/wdio-runner/src/index.ts [hunk 2]
```diff
@@ -102,6 +102,9 @@ export default class Runner extends EventEmitter {
...
+                log.error('Path to "tsconfig.json" is incorrect. Is it correclty set in config -> tsNodeOpts ? ')
```
Comment (christian-bromann) [inline packages/wdio-runner/src/index.ts:106]: Let's throw an error here because if someone really defines this property it should point to a correct file

Comment (christian-bromann) [inline packages/wdio-runner/src/index.ts:106]: Let's throw an error here because if someone really defines this property it should point to a correct file

**Review by christian-bromann — CHANGES_REQUESTED** [review]
I think this deserves a unit test.

**Comment by harsha509** [comment]
Having some wierd type issue. Will raise PR again with Test included!

**Pull request closed without merging**